.kitchen/docker_id_rsa

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
+MIIEogIBAAKCAQEAvd8GEHwt9Hs3DlEYNn3HjgUApfDuCyjtYWS0UQyqHsgZP7Uq
+eeokxNMC9ZGs1uTtqeXtmIzA+ueqcgRNK82Xo2IldNt1vYpxV3bRPHXKxzXoNyna
+VvoIi86wVlreAGQfdsds7mAxoRdc8NR6xn3HWgqykqDK40xJob6UzmbUBiLrB14f
+Yxdf2rym5jv/e64BgsCnmpVwrQDC/j5Nm6ZJRttiA2JKmZGRCv3mib4+p3hZdv05
+JPBJVBzhg1pCkyXN48wzl18ZPv9EB+dD/jnEaqh2XCDpCYYJPkjRLw827cCwCkek
+wZMdzhY5dkWxSp2wDjjTktage/l+TkZjATglNQIDAQABAoIBAHsqALUQ2H6OrtgK
+oro2PQxTEmiSMNfsZI+Y057/qBrvkfFRg4wx5kgG2RSSuptrRP/Atq8OHuCjBl2P
+jL+tTYAeWk3DTkz1UTB7Up71ieRPg6LE820YJBMMsDZUG04FpKNYGcFHhOoKTa+r
+JTk8veQarFuREgr/oYPaLIz3BvIy9YfUiCBJnuJNauFSpM5lgd7YQ+JiCXwCADQW
+zGjzw4kMS/0qbQBT0VqaQNRQ1coYWE4OjYNZDSzm5Gfj5iKfktO7y/txtsvXN736
+nkwO5pL89fqAp3Di2+xluYFVzM2JeY9klt4DcQU6eUBeZC57fDAIVXe1WassTk+Q
+t82aasECgYEA7duUrXvn7PSkeQ0Jum/XOfK9zO8xzNB6rnGOwWb/CO3tmQFzRpQb
+HtZYGbh+6gUyc63H7JT/3zYMb9Kw7NkWoW8olKlpjlf6LXey9mT6yaCp9FHhPGyQ
+EKvyLUoxTgzYmCTFOh5ftea7sX2Zv1pWHXS7+fhcWfxzvzLo5BVlCm8CgYEAzFp0
+qGhhEtz2qqDY0hhmDJ5nCn2vGz/7vtu/DpFXHCOmEi0PO0LVDQxmbgSwGEDgCr7b
+hzo+vkp07bLsgJAc6zUOkfmhm9/m6MQfUkePB7A/ZLR1O5YO2SqPc2NIAF4fc5FP
+svrptFpTUGiuCZAgAVT0xZABu/HaYWJIvH+cbJsCgYBCtS6BaRqmbgYinPF5hBxM
+eDOa4gqJcn61pctvmj80deYI30tBJBBnGZ4juIZ/g22e28ISK8eHBlV+lwxfoJKk
+O9SZvNmJvkq/ra5XRALBwXlot/3/QEM3KcIFyaUCSsAEC2v03Mfi8A77vEn+TIuS
+P0bo2CquYgrkOJbwQL/RqQKBgGGRBoWpYD+bVAHfTeXxVDfOITA3FbWS9OUJEc0o
+w0hiFNY0wULwA7x5ISaCAnlx8AXxXerQekhsUCt088nmBIigClbTsr8zRZH4oWOt
+WrK3CW1RY2+Rk4FH+rmCu0vBpI6O4Fh5x78eKQ0JDZ7vOvin0T3KJanQLQ212Th2
+TP7lAoGAY/IHN42AVG1wjrvmLjs3W2DyvYEr3odzLhfQhWQB89Yqpni59N9iceMn
+DQhxeuXzXIuAnjmHMsWC0FUzBcu+9ZgHIIqZFhTsC0zo6PS9gbgs+jKEG3U0j5hq
+pwR7fKFYqY4er3ZILQDzG01cmwndAoU5Fj/Z4e816b8vxtEPa5w=
+-----END RSA PRIVATE KEY-----
```

Comment:
doesn't kitchen coins a dummy private/pub ssh key on each run.. this is a bad missed gitignore thing